src/Keep.sol

```diff
@@ -217,7 +216,7 @@ contract Keep is ERC1155TokenReceiver, KeepToken, Multicallable {
 
             previous = signer;
 
-            emit TransferSingle(tx.origin, address(0), signer, SIGNER_KEY, 1);
+            emit TransferSingle(msg.sender, address(0), signer, SIGNER_KEY, 1);
```

Comment:
So, I was thinking here that if we use tx.origin, this logs the deployer ("summoner") of org because msg.sender would be factory otherwise?

Reply:
Yeah I guess that could be true. 
EIP1155 specifies: "The `_operator` argument MUST be the address of an account/contract that is approved to make the transfer (SHOULD be msg.sender).", but I guess it doesn't have to be `msg.sender`.

Reply:
cool, this was what I was hoping in looking at EIP. In most ERC155 implementations, the operator is logged as the mint caller, but in cases where mint is initiated through a factory, this caller info is less useful to apps, and I think origin logs the intention of getting operator 👍